core-bundle/src/Resources/public/core.js

```diff
@@ -906,15 +906,6 @@ var Backend =
 		textarea.setProperty('wrap', status);
 	},
 
-	/**
-	 * Toggle the synchronization results
-	 */
-	toggleUnchanged: function() {
-		$$('#result-list .tl_confirm').each(function(el) {
-			el.toggleClass('hidden');
-		});
-	},
-
```

Comment:
Seems to be unrelated? On purpose?

Reply:
nah, I just forgot it in https://github.com/contao/contao/pull/4327 and this was the next PR related to JS 😆